src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,75 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
```

Comment:
I agree that the tabs are quite inflexible, but they would work for this case. The loss of a consistent page framing still bothers me and I guess that removing the header from the context view was not what most other participants of the original discussion envisioned. I'll generate screenshots of the variants for comparison.